### ISSUE TITLE
Document metioned in README is missed

### ISSUE THREAD

**Issue by myungwoo** [wookayin/spotboard#2]
https://github.com/wookayin/spotboard/blob/master/docs/webapp.md

https://github.com/wookayin/spotboard/blob/master/docs/feedserver.md

Two documents mentioned in README is missed.

**Comment by wookayin** [comment]
Yeah, the document is missing as of now. I need to update the docs soon; as of now, the link is now broken. Sorry for the inconvenience.

**Comment by saadtaame** [comment]
Any updates regarding the documentation ? The link is still broken.
